osm2streets/src/geometry/mod.rs

```diff
@@ -82,6 +83,8 @@ impl InputRoad {
 pub struct Results {
     pub intersection_id: IntersectionID,
     pub intersection_polygon: Polygon,
+    /// The only transformation to `center_line` passed in must be to trim it (reducing the length)
```

Comment:
I think you were arguing for this interface to return the trim distances directly. We could clean that up as another step too

Reply:
... and I've made this change now, to save duplicating logic in A/B Street. :)